softgroup/services/main-router/src/main/java/com/softgroup/main/router/configuration/MainRouterAppCfg.java

```diff
@@ -0,0 +1,14 @@
+package com.softgroup.main.router.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 28.02.17.
+ */
+
+@Configuration
+@ComponentScan("com.softgroup")
```

Comment:
@Dominys need this scanning to check if spring properly make dependency injection for MainRouter (fills Factory<> and other...)